http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/app/CromwellRunnerAppInstallSpec.scala

```diff
@@ -69,6 +77,49 @@ class CromwellRunnerAppInstallSpec extends BaseAppInstallSpec {
       s"sam.acrPullActionIdentityResourceId=spend-profile"
   }
 
+  it should "build cromwell-runner override values for a limited bp" in {
+    val params = buildHelmOverrideValuesParams(cromwellRunnerAzureDatabases)
+    val bpid = UUID.randomUUID().toString
+    val updatedParams = params.copy(billingProfileId = BillingProfileId(bpid))
+
+    val overrides = cromwellRunnerAppInstall.buildHelmOverrideValues(updatedParams)
+
+    overrides.unsafeRunSync()(cats.effect.unsafe.IORuntime.global).asString shouldBe
+      "config.resourceGroup=mrg," +
+      "config.batchAccountKey=batchKey," +
+      "config.batchAccountName=batch," +
+      "config.batchNodesSubnetId=subnet1," +
+      s"config.drsUrl=${ConfigReader.appConfig.drs.url}," +
+      "config.landingZoneId=5c12f64b-f4ac-4be1-ae4a-4cace5de807d," +
+      "config.subscriptionId=sub," +
+      s"config.region=${azureRegion}," +
+      "config.applicationInsightsConnectionString=applicationInsightsConnectionString," +
+      "relay.path=https://relay.com/app," +
+      "persistence.storageAccount=storage," +
+      "persistence.blobContainer=sc-container," +
+      "persistence.leoAppInstanceName=app1," +
+      s"persistence.workspaceManager.url=${ConfigReader.appConfig.azure.wsm.uri.renderString}," +
+      s"persistence.workspaceManager.workspaceId=${workspaceId.value}," +
+      s"persistence.workspaceManager.containerResourceId=${storageContainer.resourceId.value.toString}," +
+      "workloadIdentity.serviceAccountName=ksa-1," +
+      "identity.name=mi-1," +
+      "cromwell.enabled=true," +
+      "fullnameOverride=cra-rel-1," +
+      "instrumentationEnabled=false," +
+      s"provenance.userAccessToken=${petUserInfo.accessToken.token}," +
+      "postgres.podLocalDatabaseEnabled=false," +
+      s"postgres.host=${lzResources.postgresServer.map(_.name).get}.postgres.database.azure.com," +
+      "postgres.pgbouncer.enabled=true," +
+      "postgres.user=ksa-1," +
+      s"postgres.dbnames.cromwell=$cromwellAzureDbName," +
+      s"postgres.dbnames.tes=$tesAzureDbName," +
+      s"postgres.dbnames.cromwellMetadata=$cromwellMetadataAzureDbName," +
+      s"ecm.baseUri=https://externalcreds.dsde-dev.broadinstitute.org," +
+      s"sam.baseUri=https://sam.test.org:443," +
+      s"sam.acrPullActionIdentityResourceId=$bpid," +
+      s"config.maxConcurrentWorkflows=100"
+  }
+
```

Comment:
looks like a lot of copy/paste with the previous test; could you extract an "expected" override string for use in both tests, then append "config.maxConcurrentWorkflows=100" for this test?